src/screens/WasteCollectionScreen.js

```diff
@@ -349,10 +365,16 @@ export const WasteCollectionScreen = ({ navigation }) => {
             data={filteredStreets}
             disableFullscreenUI
             flatListProps={{
-              renderItem: renderSuggestion
+              renderItem: renderSuggestion,
+              height: dimensions.height - normalize(220)
```

Comment:
you wrote

> - added `height` property to `flatListProps` to display all elements in the list and make
  the list scrollable

but why `dimensions.height - normalize(220)`? this is not described in the commit.

Reply:
I set it to `normalise(220)` so that the list does not overflow the bottom of the screen.

Reply:
I can't say it's perfect, but we can see the full list on the screen.

|iPad|iPhone 14|iPhone SE|iPhone 14 Pro Max|
|--|--|--|--|
![Simulator Screenshot - iPad mini (6th generation) - 2023-06-14 at 14 18 07](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/1b399301-1ffb-4cbd-856d-7c07cf3f2e27) | ![Simulator Screenshot - iPhone 14 - 2023-06-14 at 14 18 08](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/17b43e15-fdff-4c86-8489-60ed223ec15b) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2023-06-14 at 14 18 09](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/005327f8-4274-48af-998d-085fccc09d0c) | ![Simulator Screenshot - iPhone 14 Pro Max - 2023-06-14 at 14 18 11](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/659f6844-cf8f-453f-a43c-d89e52d679e5)

Reply:
ok